src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -275,6 +290,7 @@ private static void runReidPipeline(
                     .setJobName(options.getJobName())
                     .setDlpApiRetryCount(options.getDlpApiRetryCount())
                     .setInitialBackoff(options.getInitialBackoff())
+                    .setDataSinkType(options.getDataSinkType())
```

Comment:
Why are we providing this info to REID pipeline?

Reply:
we have datasink variable in DLPTransform class. So we need to set it in REID as well.